solr/solr-ref-guide/src/format-of-solr-xml.adoc

```diff
@@ -134,9 +134,10 @@ The hostname Solr uses to access cores.
 The url context path.
 
 `hostPort`::
-The port Solr uses to access cores.
+The port Solr uses to access cores, and advertise Solr node locations through liveNodes.
 +
-In the default `solr.xml` file, this is set to `${jetty.port:8983}`, which will use the Solr port defined in Jetty, and otherwise fall back to 8983.
+In the default `solr.xml` file, this is set to `${solr.port.advertise:0}`.
```

Comment:
I'd like a sentence here giving an example why the user might care about it, i.e. solr running behind a proxy.